app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb

```diff
@@ -1,4 +1,4 @@
-<p>Once you have a degree or equivalent qualification you’re ready for postgraduate primary or secondary initial teacher training courses.</p>
+<p>Once you have a degree or equivalent qualification you’re ready for postgraduate primary or secondary initial teacher training courses. You will get the teaching qualifications you need by completing your course.</p>
```

Comment:
```suggestion
<p>You'll need a degree to apply to teach primary or secondary.</p>
```
First sentence feels a bit heavy and complex? Could we just make it a lot simpler? Also feel like that second sentence is unnecessary and doesn't work - it's teaching qualification, singular, that you need (QTS), not qualifications plural. Suggest rewriting this sentence further along in the copy to integrate the keyword in a better way for the user.